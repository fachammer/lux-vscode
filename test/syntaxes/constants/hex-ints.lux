0x123456789abcdefABCDEF