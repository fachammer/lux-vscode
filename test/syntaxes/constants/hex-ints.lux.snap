>0x123456789abcdefABCDEF
#^^^^^^^^^^^^^^^^^^^^^^^ source.lux constant.numeric.int.hex.lux